anthos-bm-gcp-terraform/variables.tf

```diff
@@ -163,6 +163,12 @@ variable "abm_cluster_id" {
   default     = "cluster1"
 }
 
+variable "service_account" {
```

Comment:
```suggestion
variable "gce_vm_service_account" {
```

Reply:
Can we have this renamed to `gce_vm_service_account`? It can be confusing to just have this as `service_account` because we also have another variable called `anthos_service_account_name `

Also can we move this closer to that variable definition. Put them one after the other so the reader can quickly notice there are two and both are for different purposes